widgetastic_manageiq/__init__.py

```diff
@@ -5749,3 +5749,15 @@ def fill(self, value):
     @property
     def is_displayed(self):
         return self.textbox.is_displayed
+
+
+class HelpNavDropdown(NavDropdown):
+    @property
+    def expandable(self):
+        """Returns true with a menu can be expanded by checking the value of data-toggle"""
+        try:
+            self.browser.element('./a[contains(@data-toggle, "dropdown")]', parent=self)
```

Comment:
optional: no need to add a parent as self it will self resolve by wt.

Reply:
Correct, but not a bad thing to state explicitly. The `NavDropdown.expandable` method implementation does the same.

Overall - I would much rather see this done via a locator constant than by re-implementing the method with the exact same pattern and a different locator.

@valaparthvi Can you please open a wt.patternfly PR, updating `NavDropdown` to define and use a class attribute for this locator?  Then this class can simply define its `EXPANDABLE_LOC` in order to modify the base class.

Will look like:
```python
# in wt.patternfly:
class NavDropdown(Widget, ClickableMixin):
    EXPANDABLE_LOC = './a/span[contains(@class, "caret")]'
    # just highlighting what would change here
    @property
    def expandable(self):
        try: self.browser.element(self.EXPANDABLE_LOC, parent=self)

# in wt.manageiq
class HelpNavDropdown(NavDropdown):
    EXPANDABLE_LOC = './a[contains(@data-toggle, "dropdown")]'
```